lxd/api.go

```diff
@@ -81,7 +81,7 @@ func restServer(d *Daemon) *http.Server {
 			w.Header().Set("Permissions-Policy", "interest-cohort=()")
 			w.Header().Set("X-Content-Type-Options", "nosniff")
 			w.Header().Set("X-Frame-Options", "SAMEORIGIN")
-			w.Header().Set("X-Xss-Protection", "1; mode=block")
+			w.Header().Set("Content-Security-Policy", "default-src 'self' data: https:/*.ubuntu.com; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'")
```

Comment:
Is the `https:/` missing a forward slash?

Reply:
yes! good catch!

Reply:
fixed